forte/data/data_store.py

```diff
@@ -559,11 +605,12 @@ def _add_entry_raw(
 
         Raises:
             KeyError: raised when the entry type name is not in `self.__elements`.
+            NotImplementedError: raised when add the entry is not implemented.
```

Comment:
Well, this sentence is grammatically weird, maybe:
```suggestion
            NotImplementedError: raised when the entry type being added is not supported, currently supporting `Annotation`, `AudioAnnotation`, `Link`, `Group`, `Generics`
```

Btw, how about `Query`: https://github.com/asyml/forte/blob/master/forte/data/ontology/top.py#L38

I think we need to support all the `top` types.

Reply:
I see, you are probably right.